### ISSUE TITLE
Update prep_riboviz.py bedtools calls to add track line to bedgraph files

### ISSUE THREAD

**Issue by mikej888** [riboviz/RiboViz#22]
One of the questions on #14 is:

> 4. The BedGraph format, BedGraph Track Format, states that:
>
> BedGraph data are preceded by a track definition line...
> 
> The track type is REQUIRED, and must be bedGraph:
> ```
> type=bedGraph
> ```
>
> This isn't defined in the RiboViz BedGraph files. Should it be?

and

> These are made by bedtools genomecov. This may not matter. Check online if others have raised this as an issue.

`bedtools` supports a `-trackline` parameter which will add this line. For example:

```bash
bedtools genomecov -ibam RiboViz-Outputs/Ubuntu18/bash/WT3AT.bam -bga -5 -trackline -strand + > WT3AT_plus.bedgraph
```
Compare to "usual" RiboViz output:
```bash
diff RiboViz-Outputs/Ubuntu18/bash/WT3AT_plus.bedgraph WT3AT_plus.bedgraph 
```
```
0a1
> track type=bedGraph
```

@ewallace Should use of this parameter be added to the vignette?

**Comment by ewallace** [comment]
Yes!

**Comment by mikej888** [comment]
Implemented in commit 4f28def76b873c484630b538f0222bc1ca273719.
